cajita/src/Cajita.hpp

```diff
@@ -60,4 +60,8 @@
 #include <Cajita_SiloParticleOutput.hpp>
 #endif
 
+#ifdef Cabana_ENABLE_ALL
```

Comment:
Not sure I understand why the load balancer is dependent on `Cabana_ENABLE_ALL`? More so a question for my understanding than anything

Reply:
It uses the external library. There's more info in #457 - basically I think this should be named `LoadBalancerALL`, but since it's the only one we have right now it's not